tests/debugpy/test_multiproc.py

```diff
@@ -472,3 +472,57 @@ def before_connect(address):
 
             backchannel.send("proceed")
             child_session.wait_for_terminated()
+
+
+@pytest.mark.parametrize("run", runners.all_launch)
+@pytest.mark.skipif(sys.platform != "win32", reason="job objects are specific to Windows")
+@pytest.mark.skipif(sys.version_info < (3,), reason="CREATE_BREAKAWAY_FROM_JOB is not available in Python 2")
```

Comment:
From what I could see in this pull request, everything seems to rely only on ctypes/windows api, and I didn't really find anything python version-dependent in the implementation, so, I didn't really understand why CREATE_BREAKAWAY_FROM_JOB is not available in Python 2, so, if that's actually the case, can you elaborate a bit to me (mostly so that I can understand it) why is that the case?

Reply:
On Windows, Python exposes those constants directly on the `subprocess` module, but for Python 2, it's limited to those that were defined in WinXP SDK - so it doesn't have this particular one. I suppose the test could just pass in the numeric value directly, but it doesn't feel to be worth the bother for Py2 specifically.

Reply:
So, amusingly enough, turns out that it's not available in <3.7 either, hence the test failures. I'll update the test accordingly.